### PR TITLE
feat(CalloutBlock): implement html-view-mode & update fondue

### PR DESCRIPTION
**Pull request by Walkeezy**
- Display html in view mode of block, instead of the RTE
- Update fondue

### CODE REVIEW

File: packages/callout-block/src/CalloutBlock.tsx [hunk 1]
```diff
@@ -90,15 +93,20 @@ export const CalloutBlock: FC<BlockProps> = ({ appBridge }) => {
...
-                    border={false}
-                    value={blockSettings.textValue}
-                    placeholder="Type your text here"
-                    designTokens={useCalloutColors(designTokens, textColor)}
-                />
+                {!isEditing ? (
+                    <div data-test-id="rte-content-html" dangerouslySetInnerHTML={{ __html: html }} />
+                ) : (
+                    <RichTextEditor
+                        data-test-id="rich-text-editor"
```
Comment (fulopdaniel) [inline packages/callout-block/src/CalloutBlock.tsx:100]: doesn't RTE has this data-test-id built-in?

**Review by fulopdaniel — APPROVED** [review]

File: packages/callout-block/src/CalloutBlock.tsx (hunk 1 above)
Comment (Walkeezy) [reply packages/callout-block/src/CalloutBlock.tsx]: Holymoly, good point, thanks!

Comment (ragi96) [reply packages/callout-block/src/CalloutBlock.tsx]: Well should we really rely on "external" test ids for tests?

File: packages/callout-block/src/CalloutBlock.spec.ct.tsx [hunk 2]
```diff
@@ -7,17 +7,27 @@ import { ICON_ASSET_ID } from './settings';
...
         mount(<CalloutBlockWithStubs />);
         cy.get(CalloutBlockSelector).should('exist');
+        cy.get(RichTextEditor).should('exist');
+    });
+
+    it('should not be able to input to a callout block when in view mode', () => {
+        const [TextBlockWithStubs] = withAppBridgeBlockStubs(CalloutBlock, {});
+
+        mount(<TextBlockWithStubs />);
+        cy.get('[data-test-id="rte-content-html"]').should('exist');
```
Comment (ragi96) [inline packages/callout-block/src/CalloutBlock.spec.ct.tsx:29]: best would be to put this also in a const, so every test id is declared

File: packages/callout-block/src/CalloutBlock.tsx (hunk 1 above)
Comment (Walkeezy) [reply packages/callout-block/src/CalloutBlock.tsx]: The `data-test-id` prop on the component currently doesn't seem to have an effect anyway, so I would keep it as it for the moment. If we update fondue in the future and the 'external' test id changes, we will notice when the tests fail. :)

File: packages/callout-block/src/CalloutBlock.spec.ct.tsx (hunk 2 above)
Comment (Walkeezy) [reply packages/callout-block/src/CalloutBlock.spec.ct.tsx]: Done 👍🏻

**Review by ragi96 — APPROVED** [review]

**Pull request merged**